src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -36,7 +36,7 @@ class MovingWindow:
     Note that a numpy ndarray is returned and thus users can use
     numpys operations directly on a window.
 
-    The window uses an ringbuffer for storage and the first element is aligned to
```

Comment:
Optionally you might want to add a new commit to add the changes related to fixing documentation in  MovingWindow

Reply:
I think it is related to considering the ring buffer implementation as internal, so I put them together on purpose.